docs/bitbucket.md

```diff
@@ -0,0 +1,88 @@
+# Samples for bitbucket pipelines
+
+This are samples for running a self-hosted renovate on bitbucket.org pipelines.
+The branches
```

Comment:
> The branches

I think this needs more text?

Reply:
🤦‍♂️ Don't remember what i want to write, so removing it
```suggestion
```